test/checkpoint/workers.c

```diff
@@ -239,56 +239,51 @@ real_worker(void)
             (void)log_print_err("session.open_cursor", ret, 1);
             goto err;
         }
-    has_cursors = true;
-
-    if (g.use_timestamps)
-        begin_cfg = "read_timestamp=1,roundup_timestamps=(read=true)";
```

Comment:
When/where do we otherwise stress `roundup_timestamps`? MongoDB uses that and we need to make sure it is stress tested. Is this moved elsewhere?

Reply:
Alright, I'll add roundup back. :)